core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -275,7 +275,7 @@ impl EthTxManager {
             );
             return Err(error);
         }
-        Ok(signed_tx.hash)
+        Ok((signed_tx.hash, send_result.is_ok()))
```

Comment:
i'm super confused by this part. 
1. other methods of the code doesn't return any errors. 

Also the previous if checks this result again and return the error, so there are no chances to have `signed_tx.hash, false`

Reply:
it's an artifact from another PR; removed